docs/components/components-overview.md

```diff
@@ -15,6 +15,7 @@ This section contains product manual content for each component in Camunda Platf
 - [Operate](operate/operate-introduction.md) - User guide for monitoring and troubleshooting your process with Operate.
 - [Optimize](optimize/what-is-optimize.md) - Detailed user guide for working with Optimize to improve your processes by identifying constraints in your system.
 - [Tasklist](tasklist/introduction-to-tasklist.md) - Documentation for working with user tasks in Tasklist.
+- [Integration Framework](integration-framework/introduction.md) - Documentation on the Camunda framework to integrate with outside systems.
```

Comment:
I think this line is not also included in the 8.0 docs. Is that intentional?

Reply:
ups - true. Thanks. Corrected